File: app/src/main/java/org/take365/Network/Models/StoryProgressModel.java
package org.take365.Network.Models;

import java.io.Serializable;

/**
 * Created by evgeniy on 08.02.16.
 */
public class StoryProgressModel implements Serializable {
    public String dateStart;
    public String dateEnd;
    public double percentsComplete;
    public boolean isComplete;
    public boolean isOutdated;
    public int passedDays;
    public int totalImages;
    public String totalImagesTitle;
    public int totalDays;
    public int delayDays;
    public String delayDaysTitle;
    public boolean delayDaysMakeSense;
}

File: app/src/main/java/org/take365/Network/Models/StoryProgressModel.kt
package org.take365.Network.Models

import java.io.Serializable

/**
 * Created by evgeniy on 08.02.16.
 */
data class StoryProgressModel(
        var dateStart: String,
        var dateEnd: String,
        var percentsComplete: Double,
        var isComplete: Boolean,
        var isOutdated: Boolean,
        var passedDays: Int,
        var totalImages: Int,
        var totalImagesTitle: String,
        var totalDays: Int,
        var delayDays: Int,
        var delayDaysTitle: String,
        var delayDaysMakeSense: Boolean
) : Serializable
